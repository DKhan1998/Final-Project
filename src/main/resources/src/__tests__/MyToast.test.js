import React from "react";
import MyToast from "../Components/MyToast"
import render from "react-test-renderer";

//snapshot test
test("it matches the snapshot", () => {
    const tree = render.create(<MyToast/>).toJSON();
    expect(tree).toMatchSnapshot();
})