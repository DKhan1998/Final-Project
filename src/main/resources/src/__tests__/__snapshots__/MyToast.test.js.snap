// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches the snapshot 1`] = `
<div
  style={null}
>
  <div
    aria-atomic="true"
    aria-live="assertive"
    className="fade toast text-white border-danger bg-danger show"
    role="alert"
  >
    <div
      className="toast-header text-white bg-danger"
    >
      <strong
        className="mr-auto"
      >
        Success
      </strong>
    </div>
    <div
      className="toast-body"
    />
  </div>
</div>
`;
